docs/LwM2M_Client/STMicroelectronics/STM32L496G-DISCOBG96.md

```diff
@@ -97,9 +97,86 @@ To connect the board:
+firmware encryption key.
+
+The **Secure Boot** (Root of Trust services) is immutable code, always executed after a system reset, that checks STM32
+static protections, activates STM32 runtime protections and then verifies the authenticity and integrity of user
+application code before every execution in order to ensure that invalid or malicious code won't be run.
+
+### Additional prerequisites
+- **STM32CubeProgrammer** installed.
+- Support for shell scripts execution (on Windows for example **Git** or **Cygwin** can be used).
+- [Python](https://www.python.org/downloads/) with the following modules: `pycryptodomex`, `ecdsa`, `numpy`, `pyelftools`.
+- Import STM32L496G-Discovery_2_Images_SBSFU and STM32L496G-Discovery_2_Images_SECoreBin projects from previously cloned repository to workspace.
+
+### Prepare binary
+
+!!! important
+    You need to follow a strict compilation order presented below.
+
+0. Compile **SECoreBin** application<br/>
+   This step is needed to create the Secure Engine core binary including all the trusted code and keys mapped inside
+   the protected environment. The binary is linked with the SBSFU code in step 2.
+
+0. Compile **SBSFU** application<br/>
+   This step compiles the SBSFU source code implementing the state machine and configuring the protections. In addition,
+   it links the code with the SECore binary generated at step 1 in order to generate a single SBSFU binary including the
+   SE trusted code.
+0. Compile **UserApp** application (set **Build configuration** to **Release**)<br/>
+   It generates:<br/>
+    - The user application binary file that is uploaded to the device using the Secure Firmware Update process <br/>
+     (`Projects/STM32L496G-DISCO/UserApp/Binary/Anjay-freertos-client-STM32L496G-BG96.sfb`).
+    - A binary file concatenating the SBSFU binary, the user application binary in clear format, and the corresponding
+     FW header <br/>
+     (`Projects/STM32L496G-DISCO/UserApp/Binary/SBSFU_Anjay-freertos-client-STM32L496G-BG96.bin`).
+
+    !!! tip
+        You can set a custom firmware version in the `Application/Inc/default_config.h` file (using `FIRMWARE_VERSION` define).
+        It will be useful when performing FOTA to distinguish the firmware images from each other.
+
+### Flash the board
+
+Use **STM32CubeProgrammer** application with `SBSFU_Anjay-freertos-client-STM32L496G-BG96.bin` file to program the board (it is advisable to perform **Full chip erase** first). You can open serial port to change default credentials in order to connect to Coiote DM.
+
+After that, you can use Coiote DM to perform firmware update with `Anjay-freertos-client-STM32L496G-BG96.sfb` file.
+
+!!! important
+    Disable Secure Protection
+
+    When flashed board with Secure Boot you will need to switch off secure protection to be able to flash the board again. To deactivate secure application please run **STM32_Programmer_CLI** (Program provided with STM32CubeProgrammer) tool with specific options:
```

Comment:
Is this also an Important part? Because it looks quite strange 
![image](https://github.com/AVSystem/developer-zone/assets/61635265/70c4b341-6f62-4a86-8e38-3ff3f1686a4f)